src/components/register/RegisterForm.tsx

```diff
@@ -29,6 +34,13 @@ const RegisterForm = () => {
         },
       });
 
+      settings.dispatch({
+        type: SettingsActionTypes.SET_TOKEN,
```

Comment:
It is named a Password in Basic Auth method
```suggestion
        type: SettingsActionTypes.SET_PASSWORD,
```